### PR TITLE
Fixed the #1777 issue with Xhamster support. It is now downloading al…

### LINKED ISSUE #1777: Xhamster photo galleries stopping download at 24 images.
* Ripme version: 1.7.93
* Java version: 8_172
* Operating system: Windows 7
* Exact URL you were trying to rip when the problem occurred: https://xhamster.com/photos/gallery/sawaki-mayumi-11023311
* Please include any additional information about how to reproduce the problem: This just started today and was working fine yesterday with 1 dropped image every few galleries with nothing changed on my end so it may have been something on xhamster's side that is causing the issue.

## Expected Behavior

Expecting all 52 images of the gallery to be ripped.

## Actual Behavior

Only 24 images are downloaded before it reports it as complete despite the gallery containing more images. Many galleries I have tried from Xhamster photo have had the same problem.

### PR DESCRIPTION
**Pull request by mukulj77**
…l the images from the album instead of 24. I've also increased the jar version to 1.7.94 to avoid any conflicts. (not sure if that is appropriate or not.)

# Category

This change is exactly one of the following (please change `[ ]` to `[x]`) to indicate which:
* [x] a bug fix (Fix #1777 )
* [ ] a new Ripper
* [ ] a refactoring
* [ ] a style change/fix
* [ ] a new feature

# Description

Please add details about your change here.

# Testing

Required verification:
* [x] I've verified that there are no regressions in `mvn test` (there are no new failures or errors).
* [x] I've verified that this change works as intended.
  * [x] Downloads all relevant content.
  * [x] Downloads content from multiple pages (as necessary or appropriate).
  * [x] Saves content at reasonable file names (e.g. page titles or content IDs) to help easily browse downloaded content.
* [x] I've verified that this change did not break existing functionality (especially in the Ripper I modified).

Optional but recommended:
* [ ] I've added a unit test to cover my change.

### CODE REVIEW

File: src/main/pom.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>groupId</groupId>
+    <artifactId>main</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    
+</project>
```
Comment (soloturn) [inline src/main/pom.xml:12]: is this change really necessary, normally there are no poms at this level? if not could you force push without it pls?

Comment (mukulj77) [reply src/main/pom.xml:12]: Apologies. I'll revert the pull-request and send another one with only code change without pom. I had done that in my fork to differentiate from your releases. But I get your point.
